mim/utils/mmpack/pack_cfg.py

```diff
@@ -0,0 +1,302 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import ast
+import copy
+import os
+import os.path as osp
+import shutil
+import signal
+import sys
+import time
+import traceback
+from datetime import datetime
+from typing import Optional, Union
+
+import requests
+from mmengine import mkdir_or_exist
+from mmengine.config import Config, ConfigDict
+from mmengine.hub import get_config
+from mmengine.registry import init_default_scope
+from mmengine.runner import Runner
+
+from .common import MODULE2GitPACKAGE, __init__str, _import_pack_str
```

Comment:
The variable name starting with double underscores is unconventional; I suggest renaming it to init_str.